phplib/utils.php

```diff
@@ -58,10 +58,22 @@ static function fetch_json($hostname,$port,$protocol,$url) {
 
 
     static function deep_ksort(&$arr) {
```

Comment:
does the API allow us here to pass in the value for sort_descending instead of reading the COOKIE inside the function? Would make it less reliant on global state and eventually easier to test.